ext/php7/php7/engine_hooks.c

```diff
@@ -1080,8 +1088,16 @@ void ddtrace_message_handler(int message, void *arg) {
 
         profiling_interrupt_function = DL_FETCH_SYMBOL(handle, "datadog_profiling_interrupt_function");
         if (UNEXPECTED(!profiling_interrupt_function)) {
-            ddtrace_log_debugf("[Datadog Trace] Profiling was detected, but locating symbol %s failed: %s\n",
-                               "datadog_profiling_interrupt_function", DL_ERROR());
+            ddtrace_log_debugf("[Datadog Trace] Profiling v%s was detected, but locating symbol failed: \n",
+                               extension->version, DL_ERROR());
+        }
+
+        datadog_php_uuid (*runtime_id)(void) = DL_FETCH_SYMBOL(handle, "datadog_profiling_runtime_id");
+        if (EXPECTED(runtime_id)) {
+            ddtrace_profiling_runtime_id = runtime_id;
+        } else {
+            ddtrace_log_debugf("[Datadog Trace] Profiling v%s was detected, but locating symbol failed: \n",
```

Comment:
This `ddtrace_log_debugf` is happening in `MINIT` -- should I be doing this? @bwoebi @krakjoe

Reply:
It's fine as long as it happens after the zai config minit, which is the case.